camel/sync/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -34,4 +35,9 @@
     <routeContextRef ref="drupalAuthentication"/>
   </camelContext>
 
+  <!-- configuration of activemq component -->
+  <bean id="activemq" class="org.apache.activemq.camel.component.ActiveMQComponent">
+    <property name="islandora.brokerUrl" value="${islandora.brokerUrl}"/>
```

Comment:
@whikloj , since brokers don't run on HTTP (TCP), should we add the prefix directly to where we are going to use this instead of allowing people to decide on the cfg? I mean, tripe store does not use http:// in ca.islandora.sync.cfg